sos/report/plugins/cephadm.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class Cephadm(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = 'CEPH distributed storage managed via cephadm'
+
+    plugin_name = 'cephadm'
+    profiles = ('storage')
+
+    packages = (
+        'cephadm'
+    )
+
+    def setup(self):
+        all_logs = self.get_option("all_logs")
+
+        if not all_logs:
+            self.add_copy_spec([
+                "/var/log/ceph/cephadm.log"
+            ])
+        else:
+            self.add_copy_spec([
+                "/var/log/ceph/cephadm*"
+            ])
```

Comment:
This is already captured by the `ceph` plugin.

Reply:
This would only be captured via the `ceph` plugin when the `ceph-common` package is installed (to trigger the ceph plugin), which might not be the case in all situations (there is no package dependency between `cephadm` and `ceph-common`. `ceph-common` would usually be installed via playbook from `cephadm-ansible`).